Lib/test/string_tests.py

```diff
@@ -1081,6 +1081,21 @@ def test_mul(self):
         # but either raises a MemoryError, or succeeds (if you have 54TiB)
         #self.checkraises(OverflowError, 10000*'abc', '__mul__', 2000000000)
 
+    def test_py3x_warnings_isinstance(self):
+        with test_support.check_py3k_warnings(("the basestring type is not supported in 3.x: "
```

Comment:
Some messages have a `:` suffix, some have `: ` and some have no colon at all. We should ensure they are unified.

Reply:
The expected string in the test may need this colon. I have made the necessary changes in terms of consistency in the actual warning code for when and where to put colons.